aptos-move/aptos-vm/src/transaction_metadata.rs

```diff
@@ -27,6 +28,8 @@ pub struct TransactionMetadata {
     pub chain_id: ChainId,
     pub script_hash: Vec<u8>,
     pub script_size: NumBytes,
+    pub entry_function_payload: Option<EntryFunction>,
+    pub multisig_payload: Option<Multisig>,
```

Comment:
what's this for?

Reply:
It's for the multisig payload type. We may also need to expose it, like the entry fun payload in order for account-abstracting multisig operations. The APIs have not been added to the `transaction_context` module yet. I am afraid that this PR is getting too large. I will add those multisig payload APIs in a separate following PR.

Reply:
I also added the multisig support into this PR.